features/bootstrap/FeatureContext.php

```diff
@@ -161,7 +161,7 @@ public function iShouldSeeAPage($title)
     public function iClickOnTheTile($idpName)
     {
         $page = $this->session->getPage();
-        $idpTileTitle = sprintf('Login with your %s identity account', $idpName);
+        $idpTileTitle = sprintf('Sign in to %s', $idpName);
```

Comment:
I'm a little concerned this will be confusing for users, since in their mind I don't think they're signing in _to_ the IDP. They are using the IDP to sign into something else.

What do you think of possibly trying to keep that sort of phrasing? Possibly it's only a meaningful confusion if we also use the shortened version of the IDP display name (e.g. "Sign in to Example"), rather than the full IDP display name (e.g. "Sign in to the Example IDP"), so maybe this phrasing is still a better way if used with "the" and the full IDP's display name.

What do you think? And please don't take this as trying to derail these fixes, just refine them.